### PR TITLE
Add Pie Insurance

### PR DESCRIPTION
**Pull request by bamajap**
## Add/Update/Remove <CompanyName>

- [x] I have read the [contributing guidelines](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md)
- [x] I agree to the [Code of Conduct](https://github.com/poteto/hiring-without-whiteboards/blob/master/CODE_OF_CONDUCT.md)
- [x] I have followed the [format](https://github.com/poteto/hiring-without-whiteboards/blob/master/CONTRIBUTING.md#format) prescribed in the contributing guidelines
- [x] (OPTIONAL) In your pull request message, add additional context on the interview process if necessary

1. Phone screen with recruiter
2. Technical screen in the form of a pair-programming session to debug and test existing code
3. Final round is a multi-round session that typically includes demonstrations for backend, frontend, and design skills (and they are collaborative in nature) but are not odd Leetcode style questions.  There is also a cultural/behavioral fit round, but that does not involve coding/design demonstrations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -656,6 +656,7 @@ Also check out [No Whiteboards](https://www.nowhiteboard.org) to search for jobs
 - [pfs](https://empleo.pfsgroup.es) | Madrid, Spain; Valencia, Spain; Lisbon, Portugal; CDMX, Mexico; Remote | We adapt each interview according to the candidate and the position. The interviews are open discussions (no quizzes) about real-world problems. 2-3 interviews max, all of them can be done remotely.
 - [Philo](https://philo.com/jobs) | San Francisco, CA; Cambridge, MA | Initial screen with deep-dive into one of your recent projects (30 mins) ; take-home coding exercise tailored your skills (4 hours); on-site with take-home code review, architecture interview, design interview and "soft skills" interview (4 hours)
 - [Phoodster](https://www.phoodster.com) | Stockholm, Sweden | Take-home exercise + on-site discussion
+- [Pie Insurance](https://pieinsurance.com/careers) | Denver, CO; Washington, D.C. | Phone; remote technical screen paired with another engineer; remote multi-round interview to demonstrate basic backend/frontend skills (no bizarre Leetcode questions), behavioral/culture fit, design skills
```
Comment (cherscarlett) [inline README.md:659]: Can you clarify what the "remote technical screen" pairing entails? And what the demonstration of "basic" skills is?

Comment (bamajap) [reply README.md:659]: Sure thing..

`remote technical screen` = you are given code (with unit tests) and asked to find the bug and resolve.

`demonstration of basic skills` =

- for backend, you are given code and asked to find the bug and fix any unit tests
- for frontend, you are provided starter code and asked to build a small React component based on the given requirements (you would need to know basic React concepts to complete the task)
- for design, you're given a set of system requirements and asked to draw out on a virtual whiteboard (think draw,io or diagrams.net) what high-level system components (and their interactions) would be needed to meet the requirements.

If you would like for me to provide these details in the ReadMe, do let me know and I will add them.  Just wasn't sure how "wordy" I needed to be with this. 😃

Comment (poteto) [reply README.md:659]: Thanks!

**Pull request merged**